components/eventing-controller/pkg/backend/sink/validator.go

```diff
@@ -2,88 +2,53 @@ package sink
 
 import (
 	"context"
```

Comment:
Replaced this file by: https://github.com/kyma-project/kyma/blob/main/components/eventing-controller/pkg/backend/sink/v2/validator.go